### ISSUE TITLE
ElasticSearch target does not work

### ISSUE THREAD

**Issue by saqwel** [EsOsO/Logging#19]
Hello.
I am trying to add logs to ElasticSearch using your module, but nothing happend. I do not get any errors and do not see new messages in ElasticSearch index.
Here is my code.
```
Set-LoggingDefaultLevel -Level 'WARNING'
Add-LoggingTarget -Name ElasticSearch -Configuration @{
    ServerName  = 'localhost'
    ServerPort  = 9200
    Index       = 'powershell-2018-05-10'
    Type        = 'doc'
    #Level       = <NOTSET>          # <Not required> Sets the logging format for this target
}

$Json = '{"source" : "Logging","datetime" : "' + (Get-Date).ToUniversalTime().ToString('yyyy-MM-ddTHH:mm:ss') + '","message" : "Add message from powershell"}'
Write-Log -Level 'WARNING' -Message $Json

Wait-Logging        # See Note
```

I have successfully sent messages with Invoke-RestMethod cmdlet with parameters from example, but can't force to work your module.

**Comment by EsOsO** [comment]
Hi @saqwel, I'm working on fixing the issue; it will be available asap

**Comment by EsOsO** [comment]
Fixed in commit 496f70cd82c33a8c6f5b4b096d4fbf481001723a

Let me know if you still have issues.
